docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -0,0 +1,115 @@
+# How to convert a changelog to a table
+
+## Context
+
+You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog stream into a table. This is broadly called *materializing* a table.
```

Comment:
Should we instead say "This is broadly called *materializing* a changelog stream into a table"? The inconsistency between "materializing a table" here and "Materializing a changelog stream" below (next subheader) is a little jarring.

Reply:
Good call. I've updated it. 👍